src/HUDL/HUDL.cpp

```diff
@@ -19,43 +20,94 @@ namespace log = EVT::core::log;
 namespace HUDL {
 HUDL::HUDL(IO::GPIO& reg_select, IO::GPIO& reset, IO::SPI& spi) : lcd(DEV::LCD(reg_select, reset, spi)) {}
 
-void HUDL::dataWrite(uint8_t data) {
-    lcd.dataWrite(data);
+void HUDL::initLCD() {
+    lcd.initLCD();
+    lcd.clearLCD();
+
+    char* titles[9] = {
+        "B Voltage",
+        "Speed",
+        "RPM",
+        "Temp 1",
+        "Temp 2",
+        "Temp 3",
+        "Status 1",
+        "Pre Stat",
+        "Torque",
+    };
+
+    setDefaultSections(titles);
```

Comment:
This is setting the section string pointers to values in the array that is above, but this array only exists in the scope of this function, so it can get overwritten with other data. You should probably make `titles` a static member of the HUDL class.